src/Extension/BetterNavigatorExtension.php

```diff
@@ -1,5 +1,18 @@
 <?php
 
+namespace JonoM\BetterNavigator\Extension;
+
+use SilverStripe\ORM\DataExtension;
+use SilverStripe\Security\Permission;
+use SilverStripe\CMS\Controllers\SilverStripeNavigator;
+use SilverStripe\Control\Director;
+use SilverStripe\Security\Member;
+use SilverStripe\Core\Config\Config;
+use SilverStripe\Control\Controller;
+use SilverStripe\View\ArrayData;
+use SilverStripe\ORM\Versioning\Versioned;
```

Comment:
This has been split off so needs to be changed to `SilverStripe\Versioned\Versioned`